styles/footer.css

```diff
@@ -0,0 +1,178 @@
+.main-footer {
+  width: 1349px;
+  height: 520px;
+  margin: auto;
+  padding: 0;
+  border-top: 3px double rgb(221, 221, 221);
+  border-bottom: 1px solid rgb(221, 221, 221);
+}
+
+.main-footer-grid {
+  width: 1200px;
+  height: 400px;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-rows: 370px;
+}
+
+.header-footer {
+  margin-left: 60px;
+  width: 1200px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo-2 {
+  width: 195px;
+  height: 26px;
+  margin: 10px 0;
+}
+
+.logo-2-p {
+  font-size: 11px;
+  color: hsl(205, 27%, 46%);
+  cursor: pointer;
+}
+
+.title h5 {
+  font-size: 13px;
+  color: hsl(0, 0%, 20%);
+  text-transform: uppercase;
+  margin: 0 0 10px;
+}
+
+.ul-footer {
+  padding: 0;
+  flex-basis: 0;
+  list-style: none;
+  align-items: initial;
+  margin: auto;
+}
+
+.ul-footer li {
+  margin: 0 0 5px;
+}
+
+.footer-ul li {
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  font-size: 11px;
+  color: hsl(0, 0%, 40%);
+  margin: 0 20px 0 5px;
+}
+
+.ul-footer li:hover {
+  text-decoration: underline;
+}
+
+.ul-footer a {
+  font-size: 14px;
+  color: black;
+  text-decoration: none;
+}
+
+.aside-footer-2 {
+  float: right;
+  border-left: 1px solid rgb(221, 221, 221);
+  padding-left: 10px;
+}
+
+.aside-footer-div a {
+  font-size: 11px;
+  color: black;
+  text-decoration: none;
+}
+
+.aside-footer-div {
+  margin-bottom: 5px;
+}
+
+.aside-footer-2 a:hover {
+  text-decoration: underline;
+}
+
+.aside-footer-2 h5 {
+  font-size: 13px;
+  color: hsl(0, 0%, 20%);
+  text-transform: uppercase;
+  margin: 0 0 10px;
+}
+
+.fas {
+  display: flex;
+}
+
+.aside-footer-2 .fas {
+  width: 10px;
+  height: 10px;
+  margin-right: 5px;
+}
+
+.aside-footer-2 .fas-1 {
+  width: 17px;
+  height: 17px;
+  margin-right: 5px;
+}
+
+.subcribe-links {
+  font-size: 14px;
+  color: black;
+  text-decoration: none;
+  font-weight: 700;
+}
+
+.icons-footer-div {
+  margin-bottom: 10px;
+}
+
+.education {
+  border-bottom: 1px solid rgb(221, 221, 221);
+  padding-bottom: 3px;
+}
+
+.footer-footer {
+  display: flex;
+  border-top: 1px solid hsl(0, 0%, 92%);
+  width: 1200px;
+  height: 56px;
+  padding: 9px 0 0;
+  text-align: center;
+  margin: auto;
+  margin-bottom: 55px;
+}
+
+.footer-footer li:hover {
+  text-decoration: underline;
+}
+
+.footer-ul {
+  display: flex;
+}
+
+.footer-final {
+  position: fixed;
+  bottom: 0;
+  display: flex;
+  flex-direction: row;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_